vcd_cli/role.py

```diff
@@ -53,17 +57,47 @@ def list_roles(ctx):
               required=False,
               metavar='<org-name>',
               help='name of the org')
-def list_rights(ctx, role_name, org_name=None):
+def list_rights(ctx, role_name, org_name):
     try:
         client = ctx.obj['client']
-        if (org_name is not None):
-            org_href = client.get_org_by_name(org_name).get('href')
-        else:
-            org_href = ctx.obj['profiles'].get('org_href')
-        org = Org(client, org_href, is_sysadmin(ctx))
+        if org_name is None:
+            org_name = ctx.obj['profiles'].get('org_in_use')
+        org = Org(client, resource=client.get_org_by_name(org_name))
         role_record = org.get_role(role_name)
         role = Role(client, href=role_record.get('href'))
         rights = role.list_rights()
         stdout(rights, ctx)
     except Exception as e:
         stderr(e, ctx)
+
+
+@role.command('create', short_help='Creates role in the specified Organization (defaults to the '
+                                   'current organization in use')
+@click.pass_context
+@click.argument('role-name',
+                metavar='<role-name>',
+                required=True)
+@click.argument('description',
+                metavar='<description>',
+                required=True)
+@click.argument('rights',
+                nargs=-1,
+                metavar='<rights>')
+@click.option('-o',
+              '--org',
+              'org_name',
+              required=False,
+              metavar='[org-name]',
```

Comment:
Just make sure that if default value of org_name is not specified in the code, and user doesn't specify it either, the auto assigned value is indeed None